src/frontend/src/styles/main.css

```diff
@@ -131,6 +131,7 @@
   --vc-brand-alt: rgba(215, 205, 235, 1);
   --vc-grey: grey;
   --vc-light-grey: #d4d8e5;
+  --vc-brand-gradient: linear-gradient(90deg, #4b19d6 0%, #890eed 100%);
```

Comment:
It looks great. But I'm absolutely hopeless with CSS so it might be good to get a second opinion form @peterpeterparker or @meodai.

Reply:
Done!

Reply:
@lmuntaner: I would use vars here: `linear-gradient(90deg, var(--vc-brand-purple), var(--vc-brand-purple-light));` and the `%` are not needed if you are going to split them evenly.

Reply:
Done! Good point, I didn't realize they were the same colors, thanks!